src/Sulu/Bundle/PageBundle/Content/Types/DateTime.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Content\Types;
+
+use PHPCR\NodeInterface;
+use Sulu\Component\Content\Compat\PropertyInterface;
+use Sulu\Component\Content\SimpleContentType;
+
+/**
+ * ContentType for DateTime.
+ */
+class DateTime extends SimpleContentType
+{
+    const format = 'Y-m-d\TH:i:s';
+
+    public function __construct()
+    {
+        parent::__construct('DateTime');
+    }
+
+    public function write(
+        NodeInterface $node,
+        PropertyInterface $property,
+        $userId,
+        $webspaceKey,
+        $languageCode,
+        $segmentKey
+    ) {
+        $value = $property->getValue();
+        if (null != $value) {
+            $value = \DateTime::createFromFormat(self::format, $value);
```

Comment:
Use `static` instead of `self`, which allows to override the value in a sub class.